src/screens/Assets/rewards/RewardListItem.js

```diff
@@ -44,18 +44,19 @@ type Props = {|
   subtitle?: ?string,
   iconUrl: ?string,
   value: BigNumber,
-  symbol: string,
+  assetSymbol: string,
+  assetAddress: string,
   chain: Chain,
   onPress?: () => mixed,
 |};
 
-function RewardListItem({ title, subtitle, iconUrl, value, symbol, onPress, chain }: Props) {
+function RewardListItem({ title, subtitle, iconUrl, value, assetSymbol, assetAddress, onPress, chain }: Props) {
   const colors = useThemeColors();
 
   const chainRates = useChainRates(chain);
   const currency = useFiatCurrency();
 
-  const fiatValue = value.times(getRate(chainRates, symbol, currency));
+  const fiatValue = value.times(getAssetRateInFiat(chainRates, assetAddress, currency));
```

Comment:
Consider adding rates util `function getAssetBalanceInFiat(balance, rates, address, currency)` that would handle edge cases like missing asset rate.

Reply:
`getAssetRateInFiat` returns `0` in case there's no rate so I guess it's fine unless I miss something here? 🤔

Reply:
y, now it will probably coalesce null to zero. IMO we should differentiate between zero value and missing value. wdyt?

Reply:
I do agree, but this involves either showing some guidance to user that cannot get rate he expects and/or hide the value, not sure 🤔